src/tests/multihost/alltests/test_multidomain.py

```diff
@@ -537,3 +537,158 @@ def test_0023_ldapldap(self, multihost, multidomain_sssd):
                     print("end time = ", endtime)
                     timer.append(endtime - starttime)
             print(timer)
+
+    @pytest.mark.tier2
+    def test_0024_bz1884196(self, multihost, multidomain_sssd):
+        """
+        @Title: IDM-SSSD-TC: ldap_provider: test_for_multidomain: Check
```

Comment:
Description says, enabled option is true in both ldap domain, however in the actual test is showing only one domain has enabled option set to true. 
Could you either fix the description or the test?